event_log_track.module

```diff
@@ -89,8 +89,7 @@ function event_log_track_insert(array &$log) {
     $log['ref_numeric'] = NULL;
   }
 
-  \Drupal::database()->merge('event_log_track')
```

Comment:
`::merge()` used to be used as a poor-man's upsert; however, given [an actual `::upsert()` method existing](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Database%21Connection.php/function/Connection%3A%3Aupsert/8.9.x)... should probably make use of it if relevant; however... given the key value isn't being provided here, there's nothing to merge/upsert... if `->fields()` could ever be passed the `lid`, could see making use of it, but yeah... weird.

Reply:
Also, noticed by @jordandukart, but this is a known issue being addressed by the community: https://www.drupal.org/project/events_log_track/issues/3094617

... do we really want to produce a fork over it?

Reply:
threw in a condition similar to the community's proposed fix and just using upsert over update